samples/getting_started/BellStates.qs

```diff
@@ -1,35 +1,37 @@
-/// # Sample
-/// Bell States
+/// # Summary
+/// Bell States sample
 ///
 /// # Description
+/// This Q# sample shows how to prepare the four different Bell states.
+///
+/// # Remarks
 /// Bell states or EPR pairs are specific quantum states of two qubits
 /// that represent the simplest (and maximal) examples of quantum entanglement.
 ///
-/// This Q# program implements the four different Bell states.
-import Std.Diagnostics.*;
-
+/// # References
+/// - [Bell state](https://en.wikipedia.org/wiki/Bell_state)
 operation Main() : (Result, Result)[] {
-    // This array contains a label and a preparation operation for each one
-    // of the four Bell states.
-    let bellStateTuples = [
-        ("|Φ+〉", PreparePhiPlus),
-        ("|Φ-〉", PreparePhiMinus),
-        ("|Ψ+〉", PreparePsiPlus),
-        ("|Ψ-〉", PreparePsiMinus)
-    ];
+    // Prepare and measure each pair. Return an array of these results.
+    [
+        PrepareAndMeasurePair(PreparePhiPlus, "|Φ+〉"),
+        PrepareAndMeasurePair(PreparePhiMinus, "|Φ-〉"),
+        PrepareAndMeasurePair(PreparePsiPlus, "|Ψ+〉"),
+        PrepareAndMeasurePair(PreparePsiMinus, "|Ψ-〉")
+    ]
+}
 
-    // Prepare all Bell states, show them using the `DumpMachine` operation
-    // and measure the Bell state qubits.
-    mutable measurements = [];
-    for (label, prepare) in bellStateTuples {
-        // Allocate the two qubits that will be used to create a Bell state.
-        use register = Qubit[2];
-        prepare(register);
-        Message($"Bell state {label}:");
-        DumpMachine();
-        measurements += [(MResetZ(register[0]), MResetZ(register[1]))];
-    }
-    return measurements;
+/// # Summary
+/// Allocates a pair of qubits, prepares them using `preparation` operation,
+/// Then measures and resets them. Returns a pair of results.
+operation PrepareAndMeasurePair(
+    preparation : Qubit[] => Unit,
+    name : String
+) : (Result, Result) {
+    use pair = Qubit[2]; // Allocate a pair of qubits
```

Comment:
in most of our existing samples we follow the pattern of having comments above the line they apply to rather than to the right. I think we should stick to this rather than having a mix of both styles.

Reply:
Although I like to place short comments at the end of a statement, I agree that most of our comments are placed before the corresponding line. I changed most comments to precede the corresponding line. Only left a few that shows the state after statement.